bokehjs/gulp/tasks/test.coffee

```diff
@@ -39,6 +50,9 @@ mocha = (options={}) ->
       process.on('SIGINT',  handler)
   )
 
+gulp.task "test:coverage", ["defaults:generate"], () ->
+  gulp.src(["./test/all.coffee"]).pipe(mocha({coverage: true}))
+
 gulp.task "test", ["defaults:generate"], () ->
   gulp.src(["./test/all.coffee"]).pipe(mocha())
```

Comment:
I think the coverage report isn't generated by the ``gulp test`` task because you're not passing ``{coverage: true}`` as options to mocha here.